proposals/0024-opacity-micromaps.md

```diff
@@ -315,21 +336,24 @@ See [Opacity Micromaps][dxr-omm] in the Raytracing spec for details.
 ### Diagnostics
 
 - Check availability-based diagnostics for each flag, including recursing
-  through DeclRefs and their initializers.
+  through DeclRefs, DeclExprRefs, and their initializers.
```

Comment:
Pretty sure I understood what you meant, but I think this is more accurate.
```suggestion
  through DeclExprRefs to Decls and their initializers.
```

Reply:
I think the LLVM type is actually DeclRefExpr, I'll use that instead.